xbmc/cores/omxplayer/OMXHelper.cpp

```diff
@@ -44,6 +44,9 @@ static bool PredicateVideoPriority(const SelectionStream& lh, const SelectionStr
 
 bool OMXPlayerUnsuitable(bool m_HasVideo, bool m_HasAudio, CDVDDemux* m_pDemuxer, CDVDInputStream* m_pInputStream, CSelectionStreams &m_SelectionStreams)
 {
+  // if no OMXPlayer acceleration then it is not suitable
+  if (!CSettings::Get().GetBool("videoplayer.useomxplayer"))
```

Comment:
I guess this is supposed to be `CSettings::SETTING_VIDEOPLAYER_USEOMXPLAYER` after #7721:
https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.cpp#L171

(While at it, `CSettings::SETTING_VIDEOPLAYER_USEMMAL` a couple lines below.)